.github/workflows/win-package-test.yml

```
@@ -197,6 +197,7 @@ jobs:
           & ${{ github.workspace }}\.github\workflows\scripts\win-test-support-bundle.ps1 -mode "${{ matrix.MODE }}" -with_fluentd "${{ matrix.WITH_FLUENTD }}"
           Test-Path -Path "$env:ProgramFiles\Splunk\OpenTelemetry Collector\agent-bundle\python\python.exe"
           Test-Path -Path "$env:ProgramFiles\Splunk\OpenTelemetry Collector\agent-bundle\collectd-python"
+          Test-Path -Path "$env:opt\opentelemetry-java-contrib-jmx-metrics.jar"
```

Looks like `Test-Path` only returns true or false, and doesn't actually fail the test: https://github.com/signalfx/splunk-otel-collector/actions/runs/5695063950/job/15437772112#step:4:1451

```suggestion
          Resolve-Path -Path "$env:ProgramFiles\Splunk\OpenTelemetry Collector\agent-bundle\python\python.exe"
          Resolve-Path -Path "$env:ProgramFiles\Splunk\OpenTelemetry Collector\agent-bundle\collectd-python"
          Resolve-Path -Path "$env:SYSTEMDRIVE\opt\opentelemetry-java-contrib-jmx-metrics.jar"
```